ui/screenshot.js

```diff
@@ -0,0 +1,51 @@
+const { chromium } = require('playwright');
+const { exec } = require('child_process');
+const fs = require('fs');
+
+const fliptAddr = process.env.FLIPT_ADDRESS ?? 'http://localhost:8080';
+
+const screenshot = async (page, name) => {
+  await page.screenshot({ path: 'screenshots/'+name });
+};
+
+const sleep = (delay) => new Promise((resolve) => setTimeout(resolve, delay));
+
+const capture = async function(name, fn) {
+  try {
+    const path = `${__dirname}/screenshot/fixtures/${name}.yml`;
+    if (fs.existsSync(path)) {
+      exec(`flipt import --address=${fliptAddr} ${path}`, (error, stdout, stderr) => {
+        if (error) {
+          console.error(`error: ${error.message}`);
+          return;
+        }
+
+        if (stderr) {
+          console.error(`stderr: ${stderr}`);
+          return;
+        }
+
+        console.log(`stdout:\n${stdout}`);
+      })
+    }
+  } catch (err) {
+    // ignore and we will just skip seeding
+    console.debug(err);
+  }
+
+  const browser = await chromium.launch({ headless: true });
+  const context = await browser.newContext({
+    viewport: { width: 1280, height: 720 },
```

Comment:
wdyt about increasing the resolution to something like `viewport: { width: 1600, height: 1200 }`

Reply:
or maybe `1440x900`? I think thats the resolution I've been using when taking screenshots in Chrome for docs images (have a custom emulated device in the browser setup)

Reply:
Ill give it a go. When I did it locally I got some rather zoomed out shots at higher res. Ill paste here what that gives me.

Reply:
![create_constraint](https://github.com/flipt-io/flipt/assets/1253326/345d2ede-cfdf-42ee-9c9d-3dec8a48490f)
Yep you're right. That is nicer. I dunnot what size I had it at before.